tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -102,6 +104,24 @@ var _ = Describe("Create machinepool",
 
 			})
 
+		It("List newly added instance-types - [id:73308]",
+			labels.Medium,
+			func() {
+				By("List the available instance-types and verify the presence of newly added instance-types")
+				newlyAddedTypes := []string{"c7a.xlarge", "c7a.48xlarge", "c7a.metal-48xl", "r7a.xlarge", "r7a.48xlarge", "r7a.metal-48xl", "hpc6a.48xlarge", "hpc6id.32xlarge", "hpc7a.96xlarge", "c7i.48xlarge", "c7i.metal-24xl", "c7i.metal-48xl", "r7i.xlarge", "r7i.48xlarge"}
```

Comment:
Are all these instance types supported for all regions? Will it fail when the case running on another region?

Reply:
All the instance types will not supports for all the regions, but the method `ListInstanceTypes()` lists  all the instance-types from every region, so the case wont fail when running on another region.